index.html

```diff
@@ -14,29 +14,386 @@
+          <div class='box'>
+            <em>Connect The Dots, Africa business digest p. 102,</em>
+            <span>Aug 2017 - b inspired</span>
+          </div>
+        </a>
+        <a class='no-style' href='https://www.forbes.com/sites/tobyshapshak/2017/06/14/innovation-prize-for-africa-names-its-2017-finalists-for-solving-africas-problems/4/#3dc8c316341f' target='_blank'>
+          <div class='box'>
+            <em>Innovation Prize For Africa Names Its 2017 Finalists For Solving Africa's Problems,</em>
+            <span> 14th Jun 2017 - Forbes</span>
+          </div>
+        </a>
+        <a class='no-style' href='http://www.myjoyonline.com/news/2017/June-16th/aif-announces-the-top-10-nominees-for-innovation-prize-for-africa-2017.php' target='_blank'>
+          <div class='box'>
+            <em>AIF announces top 10 nominees for Innovation Prize for Africa 2017,</em>
+            <span>16th Jun 2017 - Myjoyonline.com</span>
+          </div>
+        </a>
+        <a class='no-style' href=https://face2faceafrica.com/article/nzola-swasisa-lokole' target='_blank'>
+          <div class='box'>
+            <em>Congolese Author’s Device Slashes Cost of Internet Data in Rural Africa,</em>
+            <span>31st Jul 2017 - face2faceafrica.com</span>
+          </div>
+        </a>
+        <a class='no-style' href='http://innovationprizeforafrica.org/news06-14-17.html' target='_blank'>
+          <div class='box'>
+            <em>AIF announces the top 10 nominees for Innovation Prize for Africa 2017,</em>
+            <span>15th Jun 2017 - innovationprizeforafrica.org</span>
+          </div>
+        </a>
+        <a class='no-style' href='http://dailyhive.com/vancouver/microsoft-global-hackathon-internet-to-congo-2017' target='_blank'>
+          <div class='box'>
+            <em>One Microsoft Vancouver engineer is working to bring internet to the Congo,</em>
+            <span>15th Sep 2017 - dailyhive.com</span>
+          </div>
+        </a>
+      </div>
+
+      <div class="transparent-section-footer">
+        <a href="#contact" class="btn btn-lg btn-next" data-toggle="tooltip" title="next page" data-placement="left">
+          <span class="glyphicon glyphicon-menu-down"></span>
+        </a>
+      </div>
+    </div>
+
+    <div class="section" data-anchor="contact">
+      <div class="container-fluid">
+        <p class="large">Any question? Any proposal?</p>
+        <p class="medium">
+          Please email us at <a href="mailto:info@ascoderu.ca" data-toggle="tooltip" title="send us an email">info@ascoderu.ca</a>
+        </p>
+
+        <div class="row-wrap-layout">
+          <div class="text-left vertical-layout" id="members-team">
+            <div class="contact-email-block row-layout">
+              <img data-src="static/profile/nzola.png" class="img-circle" id="nzola-profile">
+              <div class="member-contact">
+                <div><b>Nzola Swasisa</b>, <em>Founder & CEO</em></div>
```

Comment:
The ampersand (&) is a special character in html (used to introduce escape sequences for special characters). For displaying a literal ampersand, use `&amp;`. [More information](https://stackoverflow.com/a/3493425/3817588).